CRITICAL - 2022-02-15 02:50:27 --> syntax error, unexpected end of file, expecting elseif (T_ELSEIF) or else (T_ELSE) or endif (T_ENDIF)
#0 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#2 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(13): view('admin_template/...', Array)
#3 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#4 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ispy_erp\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-15 02:52:13 --> Call to undefined function set_value()
#0 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include()
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#3 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(13): view('admin_template/...', Array)
#4 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ispy_erp\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-15 03:51:41 --> include_once(../../Helpers/session_helper.php): failed to open stream: No such file or directory
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(6): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include_once(.....', 'C:\\xampp\\htdocs...', 6, Array)
#1 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(6): include_once()
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#5 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(18): view('admin_template/...', Array)
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ispy_erp\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-15 03:52:11 --> include_once(../../Helpers/session_helper.php): failed to open stream: No such file or directory
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(1): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include_once(.....', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(1): include_once()
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#5 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(18): view('admin_template/...', Array)
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ispy_erp\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-15 03:52:52 --> CodeIgniter\View\View::main(): Failed opening required '../../Helpers/session_helper.php' (include_path='C:\xampp\php\PEAR')
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2022-02-15 05:16:22 --> Unknown column 'updated_at' in 'field list'
CRITICAL - 2022-02-15 05:21:28 --> Invalid file: admin_template.php
#0 C:\xampp\htdocs\ispy_erp\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('admin_template....')
#1 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template', Array, true)
#2 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(74): view('admin_template', Array)
#3 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#4 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-15 05:22:30 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(160): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 160, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(74): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-02-15 05:23:05 --> Unknown column 'updated_at' in 'field list'
CRITICAL - 2022-02-15 05:25:59 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(160): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 160, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(74): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-15 05:28:34 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(160): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 160, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(74): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-02-15 05:28:53 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-02-15 05:33:34 --> Unknown column 'updated_at' in 'field list'
CRITICAL - 2022-02-15 05:34:16 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(160): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 160, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(74): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-15 05:47:45 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(160): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 160, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(74): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-15 06:08:16 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(150): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 150, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(77): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-15 06:17:40 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(150): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 150, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(77): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-15 06:17:46 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(150): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 150, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(77): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-15 06:20:20 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(150): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 150, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(77): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-15 06:29:11 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(150): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 150, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(77): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-02-15 06:52:42 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-02-15 07:01:05 --> Unknown column 'updated_at' in 'field list'
CRITICAL - 2022-02-15 10:06:44 --> Class 'App\Entities\Model' not found
#0 C:\xampp\htdocs\ispy_erp\system\Autoloader\Autoloader.php(264): include_once()
#1 C:\xampp\htdocs\ispy_erp\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\user...')
#3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\user...')
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(19): spl_autoload_call('App\\Models\\user...')
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->getData()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-15 10:22:00 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(155): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 155, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(81): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-15 10:28:49 --> Call to undefined function set_value()
#0 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include()
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#3 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(81): view('admin_template/...', Array)
#4 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-15 10:34:50 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ispy_erp\app\Views\admin_template\users.php(155): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 155, Array)
#1 C:\xampp\htdocs\ispy_erp\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ispy_erp\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ispy_erp\system\Common.php(1089): CodeIgniter\View\View->render('admin_template/...', Array, true)
#4 C:\xampp\htdocs\ispy_erp\app\Controllers\UserController.php(81): view('admin_template/...', Array)
#5 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(825): App\Controllers\UserController->insertUser()
#6 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 C:\xampp\htdocs\ispy_erp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ispy_erp\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-02-15 10:35:18 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-02-15 10:37:23 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-02-15 10:39:43 --> Unknown column 'updated_at' in 'field list'
